cmake/modules/FindUFE.cmake

```diff
@@ -82,6 +83,13 @@ find_library(UFE_LIBRARY
     NO_DEFAULT_PATH
 )
 
+# Gather all preview features that might be there or not into a single list:
+list(APPEND UFE_PREVIEW_FEATURES v4)
+
+if (UFE_INCLUDE_DIR AND EXISTS "${UFE_INCLUDE_DIR}/ufe/batchOpsHandler.h")
+    list(APPEND UFE_PREVIEW_FEATURES BatchOps)
+endif()
```

Comment:
Using a list to store keys looks like a viable solution to handle progressively added features.

Reply:
I like it, but I'm not a CMake expert.

Reply:
I think each feature added to UFE_PREVIEW_FEATURES should be tagged with the version first. So "v4_BatchOps". This would make them easier to find and remove/modify to normal ufe ifdef once Ufe v4 is released.

Reply:
Once Ufe v4 is released we remove all these "preview" checks and replace them with just "ufe v4" check.